lib/features/composer/presentation/widgets/web/local_file_drop_zone_widget.dart

```diff
@@ -52,7 +52,7 @@ class LocalFileDropZoneWidget extends StatelessWidget {
               child: Column(
                 mainAxisSize: MainAxisSize.min,
                 children: [
-                  SvgPicture.asset(imagePaths.icDropZoneIcon),
+                  Expanded(child: SvgPicture.asset(imagePaths.icDropZoneIcon)),
```

Comment:
Does it affect the display on composer? Please check again.

Reply:
It does affect somewhat. Updated